src/transformers/models/beit/image_processing_beit_fast.py

```diff
@@ -137,6 +137,16 @@ def _preprocess(
         processed_images = torch.stack(processed_images, dim=0) if return_tensors else processed_images
         return processed_images
 
+    def _preprocess_images(
+        self,
+        images,
+        **kwargs,
+    ):
+        """Preprocesses a single image."""
```

Comment:
```suggestion
        """Preprocesses images."""
```
If you could also change the one for segmentation maps it would be great :)